### PR TITLE
Changed the `max` and `all` functions on `RecordSelector` to be public

### LINKED ISSUE #136: Change RecordSelector max and all functions to be public
The two functions `max` and `all` are currently only internally available. I think it would be good to have them publicly available.

### PR DESCRIPTION
**Pull request by brototyp**
This fixes #136

### CODE REVIEW

**Comment by brototyp** [comment]
> ⚠️	Are there any changes that should be explained in the `README.md`?

No, I don't think this is necessary in this case.

File: Cauli/Configuration/RecordSelector.swift [hunk 1]
```diff
@@ -35,7 +35,7 @@ public struct RecordSelector {
     }
 }
 
-extension RecordSelector {
+public extension RecordSelector {
```
Comment (pstued) [inline Cauli/Configuration/RecordSelector.swift:38]: Well, now the RecordSelector is public, but the `max` and `all` functions are still internal.

**Review by pstued — CHANGES_REQUESTED** [review]

File: Cauli/Configuration/RecordSelector.swift (hunk 1 above)
Comment (brototyp) [reply Cauli/Configuration/RecordSelector.swift:38]: Actually, the functions have the same visibility as the class unless differently defined. In this case they are public. Would you rather like to see them explicitly public?

Comment (pstued) [reply Cauli/Configuration/RecordSelector.swift:38]: 🤦‍♂️ your right

Comment (brototyp) [reply Cauli/Configuration/RecordSelector.swift:38]: Approved? Or do you want me to change it?

**Review by pstued — APPROVED** [review]

**Pull request merged**